content/en/docs/ops/diagnostic-tools/proxy-cmd/index.md

```diff
@@ -154,30 +154,42 @@ to send a request from the `productpage` pod to the `reviews` pod at `reviews:90
+    0.0.0.0       15006 Addr: 10.244.0.22/32:15021                       inbound|15021|mgmt-15021|mgmtCluster
+    0.0.0.0       15006 Addr: 10.244.0.22/32:9080                        Inline Route: /*
+    0.0.0.0       15006 Trans: tls; App: HTTP TLS; Addr: 0.0.0.0/0       Inline Route: /*
+    0.0.0.0       15006 App: HTTP; Addr: 0.0.0.0/0                       Inline Route: /*
+    0.0.0.0       15006 App: Istio HTTP Plain; Addr: 10.244.0.22/32:9080 Inline Route: /*
+    0.0.0.0       15006 Addr: 0.0.0.0/0                                  InboundPassthroughClusterIpv4
+    0.0.0.0       15006 Trans: tls; App: TCP TLS; Addr: 0.0.0.0/0        InboundPassthroughClusterIpv4
+    0.0.0.0       15010 App: HTTP                                        Route: 15010
+    0.0.0.0       15010 ALL                                              PassthroughCluster
+    10.100.93.102 15012 ALL                                              Cluster: outbound|15012||istiod.istio-system.svc.cluster.local
+    0.0.0.0       15014 App: HTTP                                        Route: 15014
+    0.0.0.0       15014 ALL                                              PassthroughCluster
+    0.0.0.0       15021 ALL                                              Inline Route: /healthz/ready*
+    10.111.121.13 15021 App: HTTP                                        Route: istio-ingressgateway.istio-system.svc.cluster.local:15021
+    10.111.121.13 15021 ALL                                              Cluster: outbound|15021||istio-ingressgateway.istio-system.svc.cluster.local
+    0.0.0.0       15090 ALL                                              Inline Route: /stats/prometheus*
+    10.111.121.13 15443 ALL                                              Cluster: outbound|15443||istio-ingressgateway.istio-system.svc.cluster.local
     {{< /text >}}
 
-1. From the above summary you can see that every sidecar has a listener bound to `0.0.0.0:15006` which is where IP tables routes all inbound pod traffic to and a listener bound to `0.0.0.0:15001` which is where IP tables routes all outbound pod traffic to. This listener has `useOriginalDst` set to true which means
-it hands the request over to the listener that best matches the original destination of the request.
-If it can't find any matching virtual listeners it sends the request to the `PassthroughCluster` which connects to the destination directly.
+1. From the above summary you can see that every sidecar has a listener bound to `0.0.0.0:15006` which is where IP tables routes all inbound pod traffic to and a listener bound to `0.0.0.0:15001` which is where IP tables routes all outbound pod traffic to. The `PassThroughCluster` is the default cluster traffic is directed to if no listeners, routes, clusters are found during the lookup. A `Route` is both a specification of how to match a request as well as an indication of what to do next (e.g., redirect, forward, rewrite, etc.).
```

Comment:
Talking about `Route` seems out of place in the description of the following example.
```suggestion
1. From the above summary you can see that every sidecar has a listener bound to `0.0.0.0:15006` which is where IP tables routes all inbound pod traffic to and a listener bound to `0.0.0.0:15001` which is where IP tables routes all outbound pod traffic to. The `0.0.0.0:15001` listener hands the request over to the virtual listener that best matches the original destination of the request, if it can find a matching one. Otherwise, it sends the request to the `PassthroughCluster` which connects to the destination directly.
```